src/wallet/components/GoToWalletDashboard.tsx

```diff
@@ -0,0 +1,16 @@
+import { cn, pressable, text } from '../../styles/theme';
+import { walletSvg } from './walletSvg';
+
+export function GoToWalletDashboard() {
+  return (
+    <a
```

Comment:
I don't think this should be his own component. But instead have something like
```ts
<WalletDropdownItem>
  // add your own links or text
</WalletDropdownItem>
```

Mostly because I want us to teach how to create many links inside a dropdown, and avoid to force them having a special component for this.